test/jdk/java/awt/BasicStroke/TestNullShape.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/**
+ * @test
+ * @bug 6211257
+ * @summary Verifies NPE is thrown for BasicStroke.createStrokedShape(null).
+ */
+import java.awt.BasicStroke;
+import java.awt.Shape;
+
+public class TestNullShape {
+
+    public static void main(String[] args) {
+
+        BasicStroke bs = new BasicStroke();
+        try {
+            Shape s = bs.createStrokedShape(null);
+            System.out.println("result: false");
+        }
+        catch(NullPointerException ne) {
+            System.out.println("result (npe): true");
+            return;
+        }
+        throw new RuntimeException("NPE is expected");
```

Comment:
I would suggest moving this into the try-block as it would make the intention clearer: if NPE is not thrown, it's a failure.

Also catch should be on the same line as the closing brace of try.